src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/metric/temperature/capacity/ShardTotalDiskUsageTemperatureCalculator.java

```diff
@@ -0,0 +1,15 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.metric.temperature.capacity;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.temperature.TemperatureVector;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.metric.temperature.capacity.calculators.TotalNodeTemperatureCalculator;
+
+/**
+ * This class calculated the total disk used by shards in the node.
+ */
+
+public class ShardTotalDiskUsageTemperatureCalculator extends TotalNodeTemperatureCalculator {
```

Comment:
Fro every new dimension we add, we have to add 4 new graph nodes that translates into multiple Java classes. It would be nice if we can merge them so that adding a new dimension is as simple as adding a new metric to a set that would be much cleaner. 
This refactoring can be part of a separate PR. Just wanted to bring this up.